apps/cowswap-frontend/index.html

```diff
@@ -77,5 +77,6 @@
     </noscript>
     <div id="root"></div>
     <script type="module" src="/src/main.tsx"></script>
+    <script src="https://layer3.xyz/embed.js"></script>
```

Comment:
is there a particular reason the script is being added here instead of in the in the `<head></head>` tag?

Reply:
https://hackernoon.com/how-to-add-script-tags-in-react

Reply:
In the head might be blocking unless i make it deferred. I guess is possible in this case. I will experiment later to make it deferred, for now i just added using Helmet